### PR TITLE
65906 Create VUE example project

### PR DESCRIPTION
**Pull request by hannasukharieva**
Demonstrate use of Link SDK within default Vue app following the documentation on docs.codat.io

image

image

### CODE REVIEW

File: examples/vue/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <link rel="icon" href="/favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Vite App</title>
```
Comment (rodages) [inline examples/vue/index.html:7]:
```suggestion
    <title>Link SDK X Vue</title>
```

File: examples/vue/src/components/CodatLinkComponent.vue
Comment (rodages) [inline examples/vue/src/components/CodatLinkComponent.vue:1]: Should we add this to snippets folder? @mcclowes @lucy-m

File: examples/vue/README.md
Comment (rodages) [inline examples/vue/README.md:1]: can add it via my PR and did not mention this in the ticket, but would also have to update https://github.com/codatio/sdk-link/tree/main/examples to link to existing example

File: examples/vue/src/components/CodatLinkComponent.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+<template>
+  <div class="wrapper" ref="target"></div>
+</template>
+
+<style scoped>
+.wrapper {
+  width: 460px;
+  height: 840px;
+  max-height: 95%;
```
Comment (lucy-m) [inline examples/vue/src/components/CodatLinkComponent.vue:42]: What is this max height for?

Comment (rodages) [reply examples/vue/src/components/CodatLinkComponent.vue:42]: @mcclowes Included it when doing React example

File: examples/vue/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+You need to have created a company first.
+
+See <a href="https://github.com/codatio/sdk-link/tree/main#create-a-new-company" target="_blank">readme guide</a> on creating a new company.
+
+## Get started with Vue
+
+For full instructions on getting started, see our [embedded link documentation](https://docs.codat.io/auth-flow/authorize-embedded-link#get-started).
+
+1. **Create a component that mounts the SDK.** See the <a href="./src/components/">components</a> folder for an example module.
```
[the remark this answers is not in the document]
Comment (hannasukharieva) [reply examples/vue/README.md:20]: Yep, I thought it was a bit strange but assumed it was a synonym for component. Good to know, fixed!

File: examples/vue/index.html (hunk 1 above)
Comment (hannasukharieva) [reply examples/vue/index.html]: Good shout, thanks. Fixed that!

File: examples/vue/README.md
Comment (hannasukharieva) [reply examples/vue/README.md:1]: Thanks for calling this out, I added the link.

File: examples/vue/src/components/CodatLinkComponent.vue (hunk 2 above)
Comment (hannasukharieva) [reply examples/vue/src/components/CodatLinkComponent.vue:42]: We have it in all examples, might keep for consistency
image

File: examples/vue/src/components/CodatLinkComponent.vue
Comment (lucy-m) [reply examples/vue/src/components/CodatLinkComponent.vue:1]: I think it makes sense to leave this here and tell people to refer to this file in the readme.

**Review by lucy-m — APPROVED** [review]

**Pull request merged**